### PR TITLE
Copy to clipboard: strip the dollar-sign prefix from single-line shell commands.

### LINKED ISSUE #4716: Copying commands should not copy leading "$"
Example link : [https://pub.dev/packages/url_launcher/install](https://pub.dev/packages/url_launcher/install)

On copying the install command to clipboard, the leading dollar sign is also copied. This is inconvenient, because when pasting it into the shell, you need to go to the beginning of command and remove it.

On the example, clicking the copy but
```
 $ flutter pub pub add url_launcher
```
while it should be this
```
flutter pub pub add url_launcher
```

This minor enhancement will improve user experience.

### PR DESCRIPTION
**Pull request by isoos**
Fixes #4716.

### CODE REVIEW

File: pkg/web_app/lib/src/hoverable.dart [hunk 1]
```diff
@@ -108,7 +108,12 @@ void _setEventForPreCodeCopyToClipboard() {
     container.append(feedback);
 
     button.onClick.listen((_) async {
```
Comment (jonasfj) [inline pkg/web_app/lib/src/hoverable.dart:110]: This seems like the wrong place to fix this..

Should we perhaps just remove `$ ` from the template instead?

Or we could add another data property like `data-text-to-copy="..."`, but magically deciding that texts starting with `$ ` should get special treatment seems liable to introduce other bugs in the future.

Comment (isoos) [reply pkg/web_app/lib/src/hoverable.dart:110]: > Should we perhaps just remove $ from the template instead?

`dart.dev` has the same pattern for shell commands, I think we should follow that.

Comment (isoos) [reply pkg/web_app/lib/src/hoverable.dart:110]: Updated to use data- attribute, PTAL.

File: app/lib/frontend/templates/views/pkg/install_tab.mustache [hunk 2]
```diff
@@ -19,11 +19,11 @@
 <p>Run this command:</p>
 {{#use_pub_get}}
 <p>With Dart:</p>
-<pre><code class="language-shell"> $ dart pub add {{ package }}</code></pre>
+<pre data-text-to-copy="dart pub add {{ package }}"><code class="language-shell"> $ dart pub add {{ package }}</code></pre>
 {{/use_pub_get}}
 {{#use_flutter_packages_get}}
 <p>With Flutter:</p>
-<pre><code class="language-shell"> $ flutter pub pub add {{ package }}</code></pre>
+<pre data-text-to-copy="flutter pub pub add {{ package }}"><code class="language-shell"> $ flutter pub pub add {{ package }}</code></pre>
```
Comment (jonasfj) [inline app/lib/frontend/templates/views/pkg/install_tab.mustache:26]: `fluttee pub add` should work now... No need for pub pub

**Review by jonasfj — APPROVED** [review]

File: app/lib/frontend/templates/views/pkg/install_tab.mustache (hunk 2 above)
Comment (isoos) [reply app/lib/frontend/templates/views/pkg/install_tab.mustache]: Updated.

**Review by jonasfj — APPROVED** [review]

**Pull request merged**